enhancements/kube-apiserver/encrypting-data-at-datastore-layer.md

```diff
@@ -270,6 +270,30 @@ openshift-apiserver:
 
 Note: configmaps don't seem to be security-sensitive, but we know that large users of etcd encryption do encrypt them because separation of sensitive and non-sensitive data is not always easy in practice.
 
+#### Backup/Restore
+
+A backup consists of the steps 
+
+1. backup the etcd snapshot
+2. backup the master file `/etc/kubernetes/static-pod-resources/kube-apiserver-pod-<REVISION>/secrets/encryption-config/encryption-config
+` and mounted via host mount as `/etc/kubernetes/static-pod-resources/secrets/encryption-config/encryption-config` in the kube-apiserver pod.
+
+A restore must put both parts of the backup in place in master file-system before starting up kube-apiserver. 
+
+Note: the restore of the openshift-apiserver encryption config will be automatic if both (1) and (2) match.
+
+The backup of (2) must happen shortly after (1) (before another write-key is set, which happens once per week): assume (1) happens
+
+a. outside of migration: then (2) can happen
+   - before migration too
```

Comment:
outside but before? not sure if I understand the difference between `a` and `b`

Reply:
a) 
* `1--2--(migration)`
* `1--(-2-migration)`
* `1--(migration)-2--`
b) 
* `(--1--2--migration)`
* `(--1--migration)-2--`

Reply:
much better, thx :)